client/src/components/useJurisdictions.ts

```diff
@@ -144,6 +157,24 @@ export const useJurisdictions = (
   })
 }
 
+export const jurisdictionsWithLastLoginQueryKey = (
+  electionId: string
+): string[] => ['elections', electionId, 'jurisdictions', 'last-login']
```

Comment:
I think we should consider using the pattern established below where we concat to the end of ` jurisdictionsQueryKey(electionId)`. That way we won't need to explicitly invalidate this query, since it will be invalidated whenever the jurisdictions query is invalidated (since invalidating a query key will invalidate any query whose key starts with the invalidated key)